### ISSUE TITLE
Using settings.json?

### ISSUE THREAD

**Issue by SachaG** [AdmitHub/meteor-buildpack-horse#135]
I didn't see anywhere in the docs that mentions how to use a settings.json file?

From a bit of googling it looks like this works:

```
heroku config:add METEOR_SETTINGS="$(cat settings.json)"
```

Should I make a PR to add it to the docs? or is there a better way?

**Comment by yourcelf** [comment]
Good point.  I don't know of any other way to get settings than the `METEOR_SETTINGS` environment variable.

The use of `METEOR_SETTINGS` is really part of the [Meteor docs](https://docs.meteor.com/api/core.html#Meteor-settings), not something that's particular to this buildpack -- that's why we hadn't had docs yet.  But it is definitely part of what someone would need to deploy their code using this buildpack, and the additional detail with the heroku invocation would definitely be helpful.

Would accept a PR for a docs update to explain how to do this.

**Comment by voidale** [comment]
@SachaG thanks for pointing this out, just faced this issue :+1:

**Comment by RobertLowe** [comment]
@SachaG @yourcelf I maintain an old fork for some 1.2.1 apps using heroku review apps, but I found it useful to prefer `METEOR_SETTINGS` (for produciton/staging/etc) and fallback to `settings.dev.json` when `METEOR_SETTINGS` are missing, mostly used for building review apps

Food for though 👍  https://github.com/takecasper/meteor-buildpack-horse/blob/master/bin/compile#L118-L137

**Comment by stolinski** [comment]
Thank you @SachaG ! I needed this.
